hummingbot/connector/exchange/trading212/trading212_utils.py

```
@@ -0,0 +1,39 @@
+
+def is_exchange_information_valid(exchange_info: Dict[str, Any]) -> bool:
+    # Trading212 instruments array; all are tradeable for enabled accounts
+    return exchange_info.get("ticker") is not None
+
+
+class Trading212ConfigMap(BaseConnectorConfigMap):
+    connector: str = "trading212"
+    trading212_api_key: SecretStr = Field(
+        default=...,
+        json_schema_extra={
+            "prompt": lambda cm: "Enter your Trading212 API key >>> ",
```

Ruff (0.13.1)

Unused lambda argument: `cm`

(ARG005)